rpc.opam

```diff
@@ -7,7 +7,7 @@ homepage: "https://github.com/mirage/ocaml-rpc"
 doc: "https://mirage.github.io/ocaml-rpc/rpc"
 bug-reports: "https://github.com/mirage/ocaml-rpc/issues"
 depends: [
-  "ocaml" {>= "4.04.0"}
+  "ocaml" {>= "4.09.0"}
```

Comment:
Hmhmm, the constraint is too up. In this context, is it possible to use `ocaml-migrate-parsetree` to keep a compatibility with older versions of OCaml? **NOTE**: we can drop support of `4.03.0` but, at least, we need to keep a support of `4.08.0`.

Reply:
It is this commit that changed the API for the types the ppx depends on : https://github.com/ocaml/ocaml/commit/e348103ab8f234ca71cc7980d7ce656f2f26ef1b

It seems to be included in Ocaml 4.08.0, but I'm gonna give it a try before pushing anything.

Reply:
It compiles well with Ocaml 4.08.0. But without making great changes to the code (if even possible, I'm nothing of an expert on that subject), I think it would be hard to achieve an even lower bound.

Reply:
Well, I just looked at ocaml-migrate-parsetree and it looks pretty cool indeed. Do you want me to integrate it in this pull request directly (but I don't know how long it will take), or can I make another pull request with the integration of ocaml-migrate-parsetree?

Reply:
I am fine with doing it in stages.
Before this can be merged, please update the `.travis.yml` and the dune configuration to have the green tick